README.md

```diff
@@ -1,10 +1,18 @@
 # website-lambda-meetup
 
 ## Overview
-Repository for Lambda functionality related to periodically fetching data from the meetup.com API.  Built in ???.
+Repository for Lambda functionality related to periodically fetching data from the meetup.com API.  Built in Node.js
 
 ## Project Setup
 
+Please review the local development environment [steps](https://github.com/ProvidenceGeeks/website-docs/wiki/Onboarding-Guide#lambda) before getting started.
+
+1. In your IDE, Import this project as a Node.jS Express App project.
```

Comment:
I think for this, it's just a matter of giving a basic command to run, like `yarn run`

Reply:
@thescientist13 
I think it does not apply now while importing. Shall I remove it and just make it as "Import the project in your IDE "  ?

Reply:
yes please.